duo.sgnl.yaml

```diff
@@ -0,0 +1,243 @@
+description: "Duo as a System of Record"
+address: "api-{{Input Required: Duo Hostname}}.duosecurity.com" # https://duo.com/docs/adminapi#api-details
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "Duo-1.0.0"
+adapterConfig: "ewogImFwaVZlcnNpb24iOiAidjEiIAp9=" # {"apiVersion": "v1"}
+auth:
+  - basic: # https://duo.com/docs/adminapi#authentication
+      username: "{{Input Required: Duo Integration Key}}"
+      password: "{{Input Required: Duo Secret Key}}"
+entities:
+  User: # https://duo.com/docs/adminapi#retrieve-users
+    displayName: User
+    externalId: User
+    description: User entity in Duo
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 300
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: alias1
+        externalId: alias1
+        type: String
+      - name: alias2
+        externalId: alias2
+        type: String
+      - name: alias3
+        externalId: alias3
+        type: String
+      - name: alias4
+        externalId: alias4
+        type: String
+      - name: created
+        externalId: created
+        type: DateTime # UNIX Timestamp
+      - name: email
+        externalId: email
+        type: String
+      - name: firstname
+        externalId: firstname
+        type: String
+      - name: last_directory_sync
+        externalId: last_directory_sync
+        type: DateTime # UNIX Timestamp
+      - name: last_login
+        externalId: last_login
+        type: DateTime # UNIX Timestamp
+      - name: lastname
+        externalId: lastname
+        type: String
+      - name: lockout_reason
+        externalId: lockout_reason
+        type: String
+      - name: realname
+        externalId: realname
+        type: String
+      - name: status
+        externalId: status
+        type: String
+      - name: user_id
+        externalId: user_id
+        type: String
+      - name: username
+        externalId: username
+        type: String
+  Group: # https://duo.com/docs/adminapi#groups
+    displayName: Group
+    externalId: Group
+    description: Group entity in Duo
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 300
+    pagesOrderedById: false
+    attributes:
```

Comment:
This duplicate key makes the YAML invalid.

To make sure that the YAML is valid and contains all the required info, let's also upload this template to 
- a YAML validator
- a SGNL Console (`dev`/`sbx`)

Reply:
I'm not seeing any duplicates, which key are you referring to?

Reply:
Sure, we can sync on this offline.

Reply:
Verified that this works with the console. Next step is to wait for the duo adapter to be merged and test before merging this catalog right?

Reply:
finished testing